### PR TITLE
add basic UI

### PR DESCRIPTION
**Pull request by ijeomaodinko**
Create a app.rb file that will serve as your console app entry-point. It should have methods that do the following:
List all books.
List all people.
Create a person (teacher or student, not a plain Person).
Create a book.
Create a rental.
List all rentals for a given person id.
In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:
Present the user with a list of options to perform.
Lets users choose an option.
If needed, ask for parameters for the option.
Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    person_type = gets.chomp.to_i
+    puts 'Name :'
+    name = gets.chomp
+    puts 'Age :'
+    age = gets.chomp.to_i
+    case person_type
+    when 1
+      puts 'Has parent permission?[Y/N]:'
+      permission = gets.chomps.capitalize
+      @people << Student.new('unknown', age, name, permission)
```
Comment (OLIPLICHE) [inline app.rb]: - [ ] Please Check your code and solve this mistake. I see a kind of typo `chomps` in line 55. kindly check well.
Capture

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    when 2
+      puts 'specialization:'
+      specialization = gets.chomp.uppercase
+      teacher << Teacher.new(specialization, age, name)
```
Comment (OLIPLICHE) [inline app.rb]: - [ ] Please Check your code and solve this mistake. 
Capture

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  end
+
+  # to create book
+  def create_book
+    print 'Title'
+    title = gets.chomp
+    print 'Author'
+    author = gets.chomp
+    @books << Books.new(title, author)
+    puts "Book created successfully \n"
```
Comment (OLIPLICHE) [inline app.rb]: - [ ] Please Check your code and solve this mistake.
- [ ] 
Capture

- [ ] Please [Open this Link to get a Hint](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/console_apps.md#console-app-entry-point). Maybe it gonna help

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @ijeomaodinko, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used (Pull Request is made from the feature branch into the dev branch) 👏

## Required Changes ♻️

- [ ] Please make sure your school library console app behaves in the same way as the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) provided for you

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    elsif person.rentals.length.zero?
+      puts "Person #{id}: #{person.name} doesn't have rentals"
+    else
+      puts 'Rentals:'
+      person.rentals.each do |rental|
+        puts "Date: #{rental.date}, Book: \"#{rental.book.title}\" by #{rental.book.author}"
+      end
+    end
+  end
+end
```
Comment (1GORDON) [inline app.rb]: Great job @ijeomaodinko 🤝

- [ ] The UI still has some issues. Kindly make sure you can select rentals efficiently.
list rentals

File: app.rb [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+    title = gets.chomp
+    print 'Author'
+    author = gets.chomp
+    new_books = Book.new(title, author)
+    @books << new_books
+    puts "Book created successfully \n"
+  end
+
+  # to create rental
+  def create_rental
```
Comment (1GORDON) [inline app.rb:69]: Great job @ijeomaodinko 🤝

- [ ] Kindly make sure the create rental method works as expected. At the moment option 5 selects books which is not the expected behavior.
create-rental

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @ijeomaodinko ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] Kindly go over the functionality of your UI, and make sure it works as intended before you submit it again. You may check out the [video](https://www.youtube.com/watch?v=vkkgrhD6aXQ&ab_channel=EmiliaAndrzejewska) provided by the previous reviewer and try to make your design behave in a similar way.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**